rightmesh/contracts/MeshCrowdsale.sol

```diff
@@ -32,6 +32,20 @@ contract MeshCrowdsale is CappedCrowdsale, Ownable {
    * @dev minimumContribution keeps track of what should be the minimum contribution required per address
    */
   uint256 public minimumContribution;
+
+  /*
+   * @dev struct to handle the minting address and amount pairs
+   */
+  struct MintingAddressAmountPair {
+    address beneficiary;
+    uint256 amount;
+  }
+
+  /*
+   * @dev variable to keep track of what token amounts to mint to what address
+   */
+  MintingAddressAmountPair[] public mintingAddressAmountPairs;
+
```

Comment:
in this case, will map be simpler for implementation? 

for example: mapping(address => amount) public mintAddressAmount;